modules/gltools/src/context/context.js

```diff
@@ -49,8 +49,10 @@ export function createGLContext(options = {}) {
     if (options.throwOnError) {
       throw new Error(message);
     }
-    return null;
+    // eslint-disable-next-line
+    return console.error(message);
```

Comment:
This looks ugly! What is the user going to do with the return value of `console.error`? Can we keep the separate return statement after the log?

Reply:
The return value definitely shouldn't change. I could imagine existing code with `if (gl === null)`.